text/0080-datetime-extension.md

```diff
@@ -140,7 +140,12 @@ The `datetime(string)` function constructs a datetime value. Like with other ext
 - `"YYYY-MM-DDThh:mm:ss(+/-)hhmm"` (With timezone offset in hours and minutes)
 - `"YYYY-MM-DDThh:mm:ss.SSS(+/-)hhmm"` (With timezone offset in hours and minutes and millisecond precision)
 
-The `datetime` type does not provide a way for a policy author to create a `datetime` from a numeric timestamp. One of the readable formats listed above must be used, instead.
+The date and time must be valid. A valid date has `MM` between `1` and `12`, and `DD` between `1` and `31`, depending on the month/year. A valid time has `hh` between `0` and `23`, `mm` between `0` and `59`, and `ss` between `0` and `59`. In other words, a leap second is not representable. The hour offset is between `0` and `11`, even though an offset value greater than `11` can be handled with modulo operations.
```

Comment:
It's still not clear to me what the issue with representing a leap second is. As long as we can map the leap second into "Cedar time," what issues arise?

Reply:
I think it's kind of hard to map time with leap seconds into Cedar time, if the latter refers to Unix timestamps. We need to know whether the leap second is legal (i.e., exists). And it complicates the arithmetic. For instance, `end_of_leap_second.durationSince(start_of_leap_second)` should evaluate to 0 or 1s?

Reply:
In Unix time, isn't the answer 0? It replays the second, effectively ignoring it.

Reply:
Yes. But it's still impossible to tell if an instant representing a leap second is valid or not.